scripts/update-data-dir.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
```

Comment:
could've easily been a shell script but felt that since we're using `fs-extra` elsewhere, might as well make this a node script to lower the barrier for future contributors